src/sailfishui/plugin/PrivacyNotice.qml

```diff
@@ -28,13 +28,17 @@ Item {
 
     property Page page
     readonly property bool pageActive: page.status === PageStatus.Active
+    property bool noticeGiven: false
```

Comment:
false is default value... on qml you can guarantee that default values are what it is in docs regardless of cpu architecture. So... Maybe like.

```property bool privacyNoticeShown```